x-pack/platform/plugins/shared/index_management/public/application/sections/home/data_stream_list/data_stream_detail_panel/data_stream_detail_panel.tsx

```diff
@@ -348,7 +357,12 @@ export const DataStreamDetailPanel: React.FunctionComponent<Props> = ({
         content: (
           <EuiLink
             data-test-subj={'indexTemplateLink'}
-            {...reactRouterNavigate(history, getTemplateDetailsLink(indexTemplateName))}
+            href={
```

Comment:
Changes of this style are triggering a page reload. I think using `application.navigateToUrl` would fix it.